c_src/mac/keyio_mac.hpp

```diff
@@ -8,12 +8,6 @@
 #include <mach/mach_error.h>
 #include <AvailabilityMacros.h>
 
-/* The name was changed from "Master" to "Main" in Apple SDK 12.0 (Monterey) */
-/* MAC_OS_X_VERSION_12_0 does not exist :( */
-#if MAC_OS_X_VERSION_MAX_ALLOWED < 120000
-    #define kIOMainPortDefault kIOMasterPortDefault
-#endif
-
```

Comment:
Why?!

This file is common across dext and kext. This would break every user of kext, without completly removing support. I would suggest simply dropping this deletion